javascript/ql/src/Declarations/SuspiciousMethodName.ql

```diff
@@ -0,0 +1,80 @@
+/**
+ * @name Suspicious method name
```

Comment:
Perhaps "Suspicious method name declaration" to specialize the name a bit more (also requires file- and id-renamings).